src/transformers/modeling_auto.py

```diff
@@ -438,6 +449,7 @@
         (ElectraConfig, ElectraForTokenClassification),
         (FlaubertConfig, FlaubertForTokenClassification),
         (FunnelConfig, FunnelForTokenClassification),
+        (MPNetConfig, MPNetForSequenceClassification)
```

Comment:
This model is in the wrong dictionary (or if it does token classification, it should be named as such).